packages/create-app/src/template/package.json

```diff
@@ -3,8 +3,8 @@
     "version": "0.0.0",
     "private": true,
     "dependencies": {
-        "@datadog/ui-extensions-react": "0.26.0",
-        "@datadog/ui-extensions-sdk": "0.26.0",
+        "@datadog/ui-extensions-react": "0.28.0",
+        "@datadog/ui-extensions-sdk": "0.28.0",
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ These versions aren't quite right. They're what we predict the new version to be once this gets released. So like, pretty sure it'll be `0.28.0`, but not 100%. We should figure out a way to automatically update them like we do for the other versions in the repo.